optimade/server/config.ini

```diff
@@ -8,6 +8,7 @@ STRUCTURES_COLLECTION = structures
 [SERVER]
 PAGE_LIMIT = 500
 DEFAULT_DB = test_server
+BASE_URL = http://localhost:5000
```

Comment:
I prefer to keep the value in `config.py` `None`, since this allows be to check whether I should use the request to retrieve the URL or not.
I don't have access to the request from within `config.py` ...

Reply:
ok - as long as the request-based method works that is perfectly fien